src/fu-config.c

```diff
@@ -416,53 +416,57 @@ fu_config_load (FuConfig *self, GError **error)
 
 	/* load the main daemon config file */
 	config_file = g_build_filename (FWUPDCONFIGDIR, "daemon.conf", NULL);
-	g_debug ("loading config values from %s", config_file);
-	if (!g_key_file_load_from_file (self->keyfile, config_file,
-					G_KEY_FILE_NONE, error))
-		return FALSE;
-
-	/* set up a notify watch */
-	file = g_file_new_for_path (config_file);
-	monitor = g_file_monitor (file, G_FILE_MONITOR_NONE, NULL, error);
-	if (monitor == NULL)
-		return FALSE;
-	g_signal_connect (monitor, "changed",
-			  G_CALLBACK (fu_config_monitor_changed_cb), self);
-	g_ptr_array_add (self->monitors, monitor);
+	if (g_file_test (config_file, G_FILE_TEST_EXISTS)) {
```

Comment:
This is too deeply indented, can you push the contents of this to a `fu_config_load_from_file()` please.

Reply:
Fixed